core/dbt/adapters/base/relation.py

```diff
@@ -179,9 +179,9 @@ def create_from_source(cls, source, **kwargs):
             schema=source.schema,
             identifier=source.identifier,
             quote_policy={
-                'database': True,
-                'schema': True,
-                'identifier': True,
+                'database': source.quoting['database'],
+                'schema': source.quoting['schema'],
+                'identifier': source.quoting['identifier'],
```

Comment:
this new version is more correct than the old one, right? it guarantees that the rendered relation will exactly match what was supplied in the source spec. i like it